pkg/kv/bulk/bulkpb/ingestion_performance_stats.go

```diff
@@ -24,12 +24,22 @@ import (
 	"github.com/cockroachdb/cockroach/pkg/util/humanizeutil"
 	"github.com/cockroachdb/cockroach/pkg/util/log"
 	"github.com/cockroachdb/redact"
+	"github.com/codahale/hdrhistogram"
 	"github.com/gogo/protobuf/proto"
 	"go.opentelemetry.io/otel/attribute"
 )
 
 var _ bulk.TracingAggregatorEvent = (*IngestionPerformanceStats)(nil)
 
+const (
+	sigFigs    = 1
+	minLatency = time.Microsecond
+	maxLatency = 100 * time.Second
```

Comment:
Over in pkg/util/metric we have:

```

var IOLatencyBuckets = staticBucketConfig{
	category:     "IOLatencyBuckets",
	min:          10e3, // 10µs
	max:          10e9, // 10s
	count:        60,
	units:        LATENCY,
	distribution: Exponential,
}

var BatchProcessLatencyBuckets = staticBucketConfig{
	category:     "BatchProcessLatencyBuckets",
	min:          500e6, // 500ms
	max:          300e9, // 5m
	count:        60,
	units:        LATENCY,
	distribution: Exponential,
}
```

I wonder if we want to make our current Latency type something like an BatchIOLatency and make the lower bound something more like 1ms.

Reply:
that sounds reasonable, changed min to 1 Millisecond.